.goreleaser.yml

```diff
@@ -13,20 +13,18 @@ builds:
       - CGO_ENABLED=0
     goos:
       - linux
-      - darwin
-      - freebsd
     goarch:
-      - 386
       - amd64
       - arm
       - arm64
-    ignore:
-      - goos: darwin
-        goarch: 386
-      - goos: freebsd
-        goarch: arm
-      - goos: freebsd
-        goarch: arm64
+      - 386
+      - mips
+      - mips64
+      - mipsle
+      - mips64le
```

Comment:
do we need this?

Reply:
No idea, I grabbed those from the agent: https://github.com/newrelic/infrastructure-agent/blob/c9042d152df97c2f87279af557a8afea4494ad6f/build/.goreleaser_linux.yml#L142